src/cypress-api/index.ts

```diff
@@ -45,13 +50,77 @@ const doArchive = async ({
     },
     allSnapshots,
     Object.fromEntries(bufferedArchiveList),
-    // @ts-expect-error will fix when Cypress has its own package
-    { ...chromaticStorybookParams, viewport: { width: 500, height: 500 } }
+    chromaticStorybookParams
   );
 };
 
-export const archiveCypress = async (params: ArchiveParams): Promise<null> => {
-  await doArchive(params);
+// using a single Watcher instance across all tests (for the test run)
+// each time a test completes, we'll save to disk whatever archives are there at that point.
+// This should be safe since the same resource from the same URL should be the same during the entire test run.
```

Comment:
Is this safe with the resource remapping stuff @jwir3 did?

Reply:
> // This should be safe since the same resource from the same URL should be the same during the entire test run.

As long as this invariant holds, then it should be fine with the resource mapping stuff. If the content of the url changes, though (the response), then it could be problematic.

Reply:
@jwir3 can you elaborate on how the resource mapping stuff would make this a problem? 

I'd be surprised if the resource (at the same URL) changed during the test run, but wanted to make sure I'm not missing something here.